app/controllers/green_list_controller.rb

```diff
@@ -89,35 +70,26 @@ def most_protected_areas
     @regionsTopCountries = Region.without_global.map do |region|
       top_countries = RegionPresenter.new(region).top_gl_coverage_countries
       
-      if region.countries.count < 10
-        # Always return an array 10 items
-        top_countries[:countries] = top_countries[:countries].in_groups_of(10, {}).flatten
+      if top_countries[:countries].count < CHART_SIZE # Always return an array 10 items
+        if top_countries[:countries].empty?
+          # Create an array of empty hashes
+          top_countries[:countries] = Array.new(CHART_SIZE, {})
+        else
+          top_countries[:countries] = top_countries[:countries].in_groups_of(CHART_SIZE, {}).flatten
+        end
       end
-
+      
```

Comment:
I would move this logic (including the CHART_SIZE=10 constant, which could also benefit of having `.freeze` added at the end), into the region presenter.